src/components/dev-hub/related-articles.js

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import dlv from 'dlv';
+import styled from '@emotion/styled';
+import ARTICLE_PLACEHOLDER from '../../images/1x/MDB-and-Node.js.png';
+import Card from './card';
+import { H4 } from './text';
+import { colorMap, screenSize, size } from './theme';
+
+const MAX_CARD_WIDTH = 270;
+
+const RelatedContainer = styled('div')`
+    background-color: ${colorMap.devBlack};
+    padding: 30px;
+    @media ${screenSize.mediumAndUp} {
+        padding: 70px ${size.xxlarge};
+    }
+`;
+
+const RelatedCards = styled('div')`
+    display: flex;
+    @media ${screenSize.upToMedium} {
+        flex-direction: column;
```

Comment:
instead of going directly from a row to a column, should the cards wrap?

Reply:
Gregg and I talked about this and it is life to just switch to column at medium for now.

Reply:
![image](https://user-images.githubusercontent.com/514026/75487402-5b409f80-597c-11ea-92c1-4e00cc4d3434.png)

Reply:
whoops